socorro/app/socorro_app.py

```diff
@@ -202,13 +207,18 @@ def _do_run(klass, config_path=None, values_source_list=None):
                 # pull in the application defaults from the 'application'
                 # configman option, once it has been defined
                 application_defaults_proxy,
+                # Get values from the app's config module
+                klass.config_module,
                 # pull in any configuration file
                 ConfigFileFutureProxy,
                 # get values from the environment
                 environment,
                 # use the command line to get the final overriding values
                 command_line
             ]
+
+            # Remove potential None values
+            values_source_list = [x for x in values_source_list if x is not None]
```

Comment:
At the top of this file and also on line 231, the comments talk about `SocorroWelcomeApp` which no longer exists.

I think we should fix the comments at the top of this file.

I think we should change the comment on line 231 to something like:

```
# FIXME(mkelly): We used to have a SocorroWelcomeApp that defined an "application" option. We no
# longer have that. This section should get reworked possibly as part of getting rid of application
# defaults.
```